plugins/editor/src/editor/Editor.cpp

```diff
@@ -1563,6 +1579,51 @@ void Editor::Impl::updateSWLastLabel(unsigned sw, const char* label)
         updateKeyswitchNameLabel();
 }
 
+void Editor::Impl::updateBackgroundImage(const char* filepath)
+{
+    const fs::path sfzFilePath = fs::u8path(currentSfzFile_);
+    const fs::path sfzDirPath = sfzFilePath.parent_path();
+    const fs::path imagePath = sfzDirPath / filepath;
```

Comment:
Some security in case of some MinGW which use ANSI path.
```suggestion
    const fs::path imagePath = sfzDirPath / fs::u8path(filepath);
```